### ISSUE TITLE
Bug: Missions not counting towards kills / collections on different planes

### ISSUE THREAD

**Issue by zdxn** [zdxn/ValkyriesFall#22]
Issue:
When on a mission, if a player changes planes they cannot complete a mission that was selected on another plane.

Resolution:
Players should be able to complete missions on any plane if the monsters names match, I understand this might be an issue around how planes will have different monsters later but currently this is an issue.

Urgency:
5

**Comment by DavePearcey** [comment]
This is down to me not having named all the monsters correctly and called them all the same for the sake of saving some time temporarily. These are, and should be, different monsters and therefore working as expected.
